.env.local

```diff
@@ -1,4 +1,9 @@
+# Database 
 DB_CONN_STRING="mongodb+srv://admin:admin64@gamingbacklog.ce5ec.mongodb.net/libraries?retryWrites=true&w=majority"
 DB_NAME="gamebacklogDB"
 LIBRARY_COLLECTION="libraries"
 GAME_INSTANCE_COLLECTION="games"
+
+# IGDB 
+CLIENT_ID="gcicsxic8po9xjyr4x2105gxm0861s"
+CLIENT_SECRET="bnsej7verrbr55dgkdxqk8aocxl2o4"
```

Comment:
Would something like [this](https://docs.github.com/en/actions/security-guides/encrypted-secrets) be useful at all for us? So that we're not storing plain secrets in the future

Reply:
yeah probably, I can look into this more

Reply:
Created https://gaming-backlog.atlassian.net/browse/GB-29